### PR TITLE
feature/MER-4009-endpoint-message-ccself-default

### PR DESCRIPTION
**Pull request by tiandavis**
To enable receiving cross app endpoint messages:
- Added ccSelf flag to respoke.messages.send method (MER-4009)

To aid consistent formatting during open source contributions:
- Added .editorconfig for consistent contribution formatting

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -797,7 +797,8 @@ function Client(options) {
         }
 
         params = _.defaults(params || {}, {
-            type: 'message'
+            type: 'message',
+            ccSelf: true
```
Comment (pho3nixf1re) [inline lib/client.js]: Need to add this flag to the documentation comment above.

**Comment by ruffrey** [comment]
after adding the docs flag (docs redeploy?), image

**Comment by pho3nixf1re** [comment]
Yes, you will need to run the docs deploy task to update the github page. Otherwise... :+1:

File: lib/client.js (hunk 1 above)
Comment (tiandavis) [inline lib/client.js]: Done.

Comment (pho3nixf1re) [inline lib/client.js]: Be sure you've deployed to the github page. I don't see it there yet.

**Comment by tiandavis** [comment]
Docs update:
image

**Comment by ruffrey** [comment]
did you lock the version of jsdoxy to 0.7.something by removing the `^` ?

**Pull request merged**